client/state/ui/notices/actions.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
I feel like `notices` should be at root level, not under `ui`. They are only incidentally ui state because we choose to render them in layout, not because it's directly state of the UI. Specially given we may just tied them as a sort of logger for other actions.